features/polyExplorer/src/components/dataRegionsLegend/dataRegionsLegend.jsx

```diff
@@ -6,29 +6,31 @@ import i18n from "../../i18n.js";
 const DataRegionsLegend = () => {
     return (
         <div className="location-legend">
-            <div className="legend">
+            <div className="leg">
                 <p className="jurisdictions-label">
                     {i18n.t("entityDetailsScreen:jurisdictions")}:
                 </p>
-                <div>
-                    <div className="circle China"></div>
-                    <p>{i18n.t("common:jurisdiction.china")}</p>
-                </div>
-                <div>
-                    <div className="circle Five-Eyes"></div>
-                    <p>{i18n.t("common:jurisdiction.fiveEyes")}</p>
-                </div>
-                <div>
-                    <div className="circle Russia"></div>
-                    <p>{i18n.t("common:jurisdiction.russia")}</p>
-                </div>
-                <div>
-                    <div className="circle EU-GDPR"></div>
-                    <p>{i18n.t("common:jurisdiction.euGdpr")}</p>
-                </div>
-                <div>
-                    <div className="circle Others"></div>
-                    <p>{i18n.t("common:jurisdiction.undisclosed")}</p>
+                <div className="legend">
+                    <div>
+                        <div className="circle China"></div>
```

Comment:
If this is in some array, can it be factored out to a loop that uses that array as source?